docs/source/basic_examples.rst

```diff
@@ -1022,6 +1022,43 @@ useful key for metal-containing molecules. You can use the
     stk.MoleculeMongoDb = _old_molecule_init
     pymongo.MongoClient = _mongo_client
 
+Creating New Topology Graphs with Existing Vertices
+===================================================
+
+The vertex classes that make up toplogy graphs in :mod:`.stk` can be
+accessed to speed up the implemention of new topology graphs.
+
+.. testcode:: creating-new-topology-graphs-with-existing-vertices
+
+    import stk
+
+    class NewCageTopology(stk.cage.Cage):
+
+        _vertex_prototypes = (
+            stk.cage.LinearVertex(0, [0, 0, 0]),
```

Comment:
Also, I think there is a type error here. the position should be a `tuple` of `float` so `(0., 0., 0.)`

https://stk.readthedocs.io/en/stable/stk.molecular.topology_graphs.topology_graph.vertex.html#stk.molecular.topology_graphs.topology_graph.vertex.Vertex

I realize that this type error exists in the actual code, but I don't want you to change that. This PR just needs to make sure that users get a fully correct example.

Reply:
Done.